ocaml/sdk-gen/component-test/spec/go_test.json

```diff
@@ -0,0 +1,877 @@
+{
+  "test_id1": {
+    "method": [
+      "session.login_with_password",
+      "pool.get_all",
+      "pool.get_record",
+      "host.get_record",
+      "session.logout"
+    ],
+    "params": {
+      "session.login_with_password": ["", "", "1.0", "Go SDK component test"],
+      "pool.get_all":  ["OpaqueRef:12345678-4d8a-e4a9-a241-1ea05c1d6707"],
```

Comment:
Could the ref name be more clarifying? Such as:
Pool: "OpaqueRef:####pool-0000-0000-0000-000000000001"
Host: "OpaqueRef:####host-0000-0000-0000-000000000001", "OpaqueRef:####host-0000-0000-0000-000000000002", etc.
VM: "OpaqueRef:######vm-0000-0000-0000-000000000001", "OpaqueRef:######vm-0000-0000-0000-000000000002", etc.

Reply:
Ref has non meaning itself, I replace some string into vm is for get different ref. So I suspect there is any need to change as above format.